java/src/test/java/scenario/ScenarioSteps.java

```diff
@@ -732,4 +753,11 @@ private static PrivateKey getPrivateKey(final Path privateKeyPath) throws IOExce
             return Identities.readPrivateKey(privateKeyReader);
         }
     }
+
+    private void assertCheckpointerExist() {
```

Comment:
I don't think the rest of the Java test implementation does explicit checking that things are non-null so I wouldn't bother doing that for this case either